Modules/BenchPress.Azure/Tests/Public/Confirm-StorageContainer.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+BeforeAll {
+  . $PSScriptRoot/../../Public/Confirm-StorageContainer.ps1
+  . $PSScriptRoot/../../Private/Connect-Account.ps1
+  Import-Module Az
+}
+
+Describe "Confirm-StorageContainer" {
+  Context "unit tests" -Tag "Unit" {
+    BeforeEach {
+      Mock Connect-Account{}
+    }
+
+    It "Calls Get-AzStorageContainer" {
+      Mock Get-AzStorageAccount{
+        $stgAcct = [Microsoft.Azure.Management.Storage.Models.StorageAccount]::new()
```

Comment:
Ideally this would just return a mock with `New-MockObject -Type "Microsoft.Azure.Commands.Management.Storage.Models.PSStorageAccount"` so that there is no dependency on any library and we're not "newing up" anything that we're not testing.

Reply:
Wow I was scouring the internet today for something like this! My searching skills need some improvement LOL Thanks for pointing me in the right direction. Implemented this change!